GME/Gme/GMEApp.cpp

```diff
@@ -124,8 +125,12 @@ BEGIN_MESSAGE_MAP(CGMEApp, CWinAppEx)
 	ON_COMMAND_EX_RANGE(ID_FILE_MRU_PRJ1, ID_FILE_MRU_PRJ16, OnOpenRecentProject)
 	ON_COMMAND_RANGE(ID_FILE_RUNPLUGIN1, ID_FILE_RUNPLUGIN_LAST, OnRunPlugin)
 	ON_COMMAND_RANGE(ID_FILE_INTERPRET1, ID_FILE_INTERPRET_LAST, OnRunInterpreter)
-	ON_UPDATE_COMMAND_UI_RANGE( ID_FILE_RUNPLUGIN1, ID_FILE_RUNPLUGIN_LAST, OnUpdateFilePluginX)
-	ON_UPDATE_COMMAND_UI_RANGE( ID_FILE_INTERPRET1, ID_FILE_INTERPRET_LAST, OnUpdateFileInterpretX)
+	ON_COMMAND(ID_HELP_COMPONENTHELP, DummyCommand)

```

Comment:
Was this change intentional?

Reply:
Yes, the "dummy command" is needed so that the popup menu can be enabled.